docs/src/library/rule.md

```diff
@@ -60,13 +60,13 @@ A rule has a:
 }) }}
 
 {{ rule({
-    name: "getter-return",
-    deprecated: false,
-    description: 'Enforce `return` statements in getters.',
-    categories: {
-        recommended: true,
-        fixable: false,
-        hasSuggestions: false
-    }
+name: "getter-return",
+deprecated: false,
+description: 'Enforce `return` statements in getters.',
+categories: {
+recommended: true,
+fixable: false,
+hasSuggestions: false
+}
```

Comment:
Can we ignore files in the `/library` folder? These are showing macro examples and the indentation is useful to have.